### DISCUSSION TITLE
Layout and Predefined Application at Autostart.

### DISCUSSION THREAD

**Discussion by dwskme** [i3/i3#4670] [Q&A]
Hey Everyone, I want my workspace to look like this.
![image](https://user-images.githubusercontent.com/55840147/141642839-3b6d0e63-38e4-4d7c-8f2b-f11700f0e0ac.png)

I want the workspace to have a master and 2 children layout. And I want them opening a terminal and different application like, neofetch, ranger, spotify-cli etc in each of them. How do i Make them?

**Comment by Airblader** [comment]
https://i3wm.org/docs/layout-saving.html
